src/main/java/org/mockito/internal/MockitoCore.java

```diff
@@ -39,6 +40,19 @@
     private final Reporter reporter = new Reporter();
     private final MockUtil mockUtil = new MockUtil();
     private final MockingProgress mockingProgress = new ThreadSafeMockingProgress();
+    private VerificationStrategy verificationState;
+
+    public MockitoCore() {
+        this.verificationState = getDefaultVerificationStrategy();
```

Comment:
I'm not sure about keeping state in Mockito core instance. Why not MockingProgress?